packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardUtils.ts

```diff
@@ -82,65 +82,44 @@ export const formatReleaseCardData = (
   }
 }
 
-const createButtonToExploreDataQueryString = (
+export const createButtonToExploreDataPathAndQueryString = (
   schema: ReleaseCardSchema,
   data: (string | null)[],
   buttonToExploreDataConfig?: ButtonToExplorePageConfig,
 ) => {
   if (!buttonToExploreDataConfig) return null
+  const path = getValueFromData(
+    schema,
+    data,
+    buttonToExploreDataConfig.sourcePathColumnName,
+  )
+  if (!path) {
+    console.warn(
+      `Column not found in source table or cell did not have value in source table for ${buttonToExploreDataConfig.sourcePathColumnName}`,
+    )
+    return null
+  }
 
   const {
     exploreDataSql,
     exploreDataFacetColumnName,
     sourceDataFacetValueColumnName,
   } = buttonToExploreDataConfig
-  if (
-    exploreDataSql &&
-    exploreDataFacetColumnName &&
-    sourceDataFacetValueColumnName
-  ) {
-    const facetValue = getValueFromData(
-      schema,
-      data,
-      sourceDataFacetValueColumnName,
-    )
-    // column not found in source table or cell did not have value in source table
-    if (facetValue === null) return null
 
-    const query: Query = {
-      sql: exploreDataSql,
-      selectedFacets: [
-        {
-          concreteType:
-            'org.sagebionetworks.repo.model.table.FacetColumnValuesRequest',
-          columnName: exploreDataFacetColumnName,
-          facetValues: [facetValue],
-        },
-      ],
-    }
-    return `?QueryWrapper0=${encodeURIComponent(JSON.stringify(query))}`
-  }
-  return null
-}
+  const facetValue = sourceDataFacetValueColumnName
+    ? getValueFromData(schema, data, sourceDataFacetValueColumnName)
+    : null
+  const hasSelectedFacet =
+    exploreDataSql && exploreDataFacetColumnName && facetValue
 
-export const createButtonToExploreDataPathAndQueryString = (
-  schema: ReleaseCardSchema,
-  data: (string | null)[],
-  buttonToExploreDataConfig?: ButtonToExplorePageConfig,
-) => {
-  if (!buttonToExploreDataConfig) return null
-  const path = getValueFromData(
-    schema,
-    data,
-    buttonToExploreDataConfig.sourcePathColumnName,
-  )
-  // column not found in source table or cell did not have value in source table
-  if (!path) return null
+  if (!hasSelectedFacet) console.log(`No selected facet for ${path}`)
```

Comment:
In the GENIE portal, release cards will either point directly to an explore page and or also include a selected facet. The source table will have `null` values in the selected facet column for the cards that should point directly to the explore page. Since this is an expected case, log a message about the missing facet rather than a warning.

Reply:
Oh sorry, I misinterpreted the last version of the code. It's much more clear now with your explanation and the changes to the logical flow here. I don't think you need to log anything, since no facets is definitely a valid case!